Java/Rahul/SelectionSort.java

```diff
@@ -0,0 +1,29 @@
+/* Online Java Compiler and Editor */
+public class SelectionSort{
+
+     public static void main(String []args){
+         int[] input = new int[]{10,3,8,9,5};
+          selectionSort(input);
+         for (int i = 0; i < input.length; i++)
+            System.out.println("Element at index " + i
+                               + " : " + input[i]);
+
+     }
+     
+     public static int[] selectionSort(int[] input){
+         
+        for (int j=0;j<input.length-1;j++){
+            int minimum = input[j];
+            int minIndx = j;
+            for(int i= j ; i < input.length-1 ;i++){
+                if(minimum > input[i+1]){
```

Comment:
why are we comparing with i+1 ? While this absolutely works fine, It is bit confusing. May be starting inner loop at j+1 will be more clearer? What do you think?

Reply:
Yes I agree
I thought that in insertionsort and changed that
Will change this as well

Reply:
changed